site/en/getstarted/run-milvus-docker/install_standalone-docker-compose.md

```diff
@@ -55,10 +55,10 @@ $ sudo docker-compose ps
 --------------------------------------------------------------------------------------------------------------------
 milvus-etcd         etcd -advertise-client-url ...   Up             2379/tcp, 2380/tcp
 milvus-minio        /usr/bin/docker-entrypoint ...   Up (healthy)   9000/tcp
-milvus-standalone   /tini -- milvus run standalone   Up             0.0.0.0:19530->19530/tcp, 0.0.0.0:9091->9091/tcp
+milvus-standalone   /tini -- milvus run standalone   Up             0.0.0.0:19530->19530/tcp, 0.0.0.0:9001->9001/tcp
 ```
 
-You can also access Milvus WebUI at `http://127.0.0.1:9091/webui/` to learn more about the your Milvus instance. For details, refer to [Milvus WebUI](milvus-webui.md).
+You can also access Milvus WebUI at `http://127.0.0.1:9001/webui/` to learn more about the your Milvus instance. For details, refer to [Milvus WebUI](milvus-webui.md).
```

Comment:
webui is exposed with 9091 port,
so the url should be `http://127.0.0.1:9091/webui`